plugins/kotlin-as-java/src/main/kotlin/converters/KotlinToJavaConverter.kt

```diff
@@ -76,7 +77,11 @@ internal fun DPackage.asJava(): DPackage {
     )
 }
 
-internal fun DProperty.asJava(isTopLevel: Boolean = false, relocateToClass: String? = null) =
+internal fun DProperty.asJava(
+    isTopLevel: Boolean = false,
+    relocateToClass: String? = null,
+    isBelongToObjectOrCompanion: Boolean = false
```

Comment:
nitpick: `isBelong..` is not grammatically correct, what do you think about `isFrom..`?